src/MoBi.Core/Serialization/Converter/v6_0/Converter36To601.cs

```diff
@@ -16,32 +16,42 @@ public class Converter36To601 : IMoBiObjectConverter,
       private readonly Converter56To601 _coreConverter56To601;
       private static readonly string RHS_DIMENSION_OLD_SUFFIX = " per Time";
       private static readonly string MOL_WEIGHT = "MolWeight";
+      private bool _converted;
 
       public Converter36To601(Converter56To601 coreConverter56To601)
       {
          _coreConverter56To601 = coreConverter56To601;
       }
 
-      public bool IsSatisfiedBy(int version)
-      {
-         return version == ProjectVersions.V3_6_1;
-      }
+      public bool IsSatisfiedBy(int version) => version == ProjectVersions.V3_6_1;
 
-      public int Convert(object objectToUpdate, IMoBiProject project)
+      public (int convertedToVersion, bool conversionHappened) Convert(object objectToUpdate, IMoBiProject project)
       {
-         _coreConverter56To601.Convert(objectToUpdate);
+         _converted = false;
+         var (_, coreConversionHappened) = _coreConverter56To601.Convert(objectToUpdate);
          this.Visit(objectToUpdate);
-         return ProjectVersions.V6_0_1;
+         return (ProjectVersions.V6_0_1, _converted || coreConversionHappened);
       }
 
-      public int ConvertXml(XElement element, IMoBiProject project)
+      public (int convertedToVersion, bool conversionHappened) ConvertXml(XElement element, IMoBiProject project)
       {
-         _coreConverter56To601.ConvertXml(element);
+         _converted = false;
+         var (_, coreConversionHappened) = _coreConverter56To601.ConvertXml(element);
          element.DescendantsAndSelfNamed("DisplayUnitMap").Each(displayUnitMapElement => convertPerTimeDimension(displayUnitMapElement, "dimension"));
          element.DescendantsAndSelfNamed("Formula").Each(displayUnitMapElement => convertPerTimeDimension(displayUnitMapElement, "dim"));
          element.DescendantsAndSelfNamed("Map").Each(displayUnitMapElement => convertPerTimeDimension(displayUnitMapElement, "s"));
          element.DescendantsAndSelfNamed("Parameter").Each(convertParameterQuantityType);
-         return ProjectVersions.V6_0_1;
+         return (ProjectVersions.V6_0_1, _converted || coreConversionHappened);
+      }
+
+      public void Visit(IMoBiProject project)
```

Comment:
moving public method to the top is a simple way to see what's going on with the component